service/src/main/java/aero/minova/cas/SecurityConfig.java

```diff
@@ -49,6 +49,7 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
 	@Override
 	protected void configure(HttpSecurity http) throws Exception {
+		http.authorizeRequests().antMatchers("/actuator/**").permitAll();
 		http.authorizeRequests().antMatchers("/", "/public/**", "/img/**", "/js/**", "/theme/**", "/index", "/login", "/layout").permitAll();
```

Comment:
Den String kannst du auch in die Zeile drunter eingfügen zu den anderen Permits.

Reply:
Wäre es möglich `/actuator/prometheus/**` abzusichern? https://stackoverflow.com/questions/60272588/securing-spring-boot-2-actuator-prometheus-end-points.

Ansonsten sollte man das neue Feature in der Changelog dokumentieren: https://github.com/minova-afis/aero.minova.cas/blob/master/CHANGELOG.adoc